### PR TITLE
Fix mutation and make parameters optional

### PR DESCRIPTION
**Pull request by simonprast**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there are static parameters used for the AlterUser mutation.
- There is a bug with the mutation method because there is the info parameter missing.

**What is the new behavior (if this is a feature change)?**
- Add info parameter to mutate method.
- The AlterUser mutation now accepts optional parameters.

### CODE REVIEW

File: charm/user/schema.py [hunk 1]
```diff
@@ -89,30 +89,32 @@ class Arguments:
         email = graphene.String(required=False)
         is_active = graphene.Boolean(required=False)
 
-    def mutation(self, user_id, first_name, last_name, city, country, address, postal_code, telephone, email, is_active):
+    def mutate(self, info, user_id, **_kwargs):
```
Comment (Aichnerc) [inline charm/user/schema.py:94]: Please add commentary about the functionality of this mutation.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
Please add the missing commentary.

File: charm/user/schema.py (hunk 1 above)
Comment (simonprast) [reply charm/user/schema.py:94]: ok.

**Review by Aichnerc — APPROVED** [review]
The commits have not been verified. Ok for now, please fix later.

Code seems to be ok.

**Pull request merged**